### PR TITLE
Fix Issue #124

### LINKED ISSUE #124: Btn/waitng issue
1. btn color change
2. add waiting icon 
3. delete "... " on detail profile page
4. Deactivate send function.

### PR DESCRIPTION
**Pull request by ifndefdeadmau5**
Fixed #124

### CODE REVIEW

File: src/components/UserList/Row.js [hunk 1]
```diff
@@ -162,6 +163,33 @@ class Row extends Component {
       },
     ];
 
+    let connectButton = null;
+
+    if (this.state.pending) {
+      connectButton = (
+        <View style={[styles.connectBtnStyle, { backgroundColor: '#a6aeae' }]}>
+          <View style={styles.buttonContainer}>
+            <View style={{ paddingTop: 10, marginRight: 5, }}>
```
Comment (MechanicKim) [inline src/components/UserList/Row.js:172]: Unexpected trailing comma

File: src/components/UserList/Row.js [hunk 2]
```diff
@@ -162,6 +163,33 @@ class Row extends Component {
...
+              <Icon name="clock-o" size={15} color="white" />
+            </View>
+            <Text style={styles.buttonText}>WAITING</Text>
+          </View>
+        </View>
+      );
+    } else {
+      connectButton = (
+        <LinearGradient style={styles.connectBtnStyle} start={[0.9, 0.5]} end={[0.0, 0.5]}
```
Comment (MechanicKim) [inline src/components/UserList/Row.js]: Property should be placed on a new line

File: src/components/UserList/Row.js [hunk 3]
```diff
@@ -162,6 +163,33 @@ class Row extends Component {
...
+                        locations={[0, 0.75]}
```
Comment (MechanicKim) [inline src/components/UserList/Row.js:184]: Expected indentation of 10 space characters but found 24

Next line(183) is same.

File: src/components/UserList/Row.js [hunk 4]
```diff
@@ -162,6 +163,33 @@ class Row extends Component {
...
+            <View style={styles.buttonContainer}>
+              <Text style={styles.buttonText}>CONNECT</Text>
+            </View>
+          </TouchableOpacity>
+        </LinearGradient>
+      );
+    }
+
     return (
         <TouchableWithoutFeedback onPress={goToUserProfile}>
```
Comment (MechanicKim) [inline src/components/UserList/Row.js:196]: Expected indentation of 6 space characters but found 8

**Comment by MechanicKim** [comment]
@ifndefdeadmau5 
I reviewed this PR.

**Comment by ifndefdeadmau5** [comment]
Thanks! I fixed JSCS warning

**Comment by MechanicKim** [comment]
@SarahhhC please review this PR.

**Comment by SarahhhC** [comment]
LGTM Nice fix!!

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/127/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**